app/src/main/res/layout/item_library.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<androidx.cardview.widget.CardView
+<androidx.cardview.widget.CardView xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
+  style="@style/CardView_style"
```

Comment:
You are still using a style and I want this to be theme driven so that by default all our cardviews have this background.

You should specify in our light theme `cardBackgroundColor` and specify a different color in our night theme.

If using `cardBackgroundColor` doesn't work you can set the style for all CardViews in the theme.

From a quick look there are 2 attributes it could be `cardViewStyle` or `materialCardViewStyle`. See `navigationViewStyle` for a template of how we set styles in a theme

Reply:
Cardview used is not material in parent. Should I change it to material one and make a theme for all our Cardviews.

Reply:
Done. Please Review